selfdrive/car/hyundai/values.py

```diff
@@ -28,7 +28,7 @@ class CAR:
   IONIQ_EV_2020 = "HYUNDAI IONIQ ELECTRIC 2020"
   KONA = "HYUNDAI KONA 2020"
   KONA_EV = "HYUNDAI KONA ELECTRIC 2019"
-  SANTA_FE = "HYUNDAI SANTA FE LIMITED 2019"
+  SANTA_FE = "HYUNDAI SANTA FE LIMITED 2019-2020"
```

Comment:
revert this, no need to update it

Reply:
has been reverted back to only 2019

Reply:
Any news on this. It has fixed my issue locally. Am I missing anything else to get this approved?